### DISCUSSION TITLE
How to run standalone app after building with npm?

### DISCUSSION THREAD

**Discussion by kneelesh48** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#244] [Q&A]
I git cloned the repository and built the app with npm and I can run it with `npm start` but I have to keep a terminal window open.

If I run download from the releases page, I just need to run Google Assistant.exe and it runs in Windows Tray.

How can I achieve the same thing with self compiled version?

**Comment by Melvin-Abraham** [comment]
@kneelesh48 Do refer [this section from CONTRIBUTING.md](https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/blob/master/CONTRIBUTING.md#how-to-build) for details regarding this.
